orchestrator-salt/src/main/resources/salt/salt/postgresql/disaster_recovery/scripts/backup_db_compressed.sh

```diff
@@ -0,0 +1,254 @@
+#!/bin/bash
+# backup_db.sh
+# This script uses the 'pg_dump' utility to dump the contents of hive and ranger PostgreSQL databases as plain SQL.
+# After PostgreSQL contents are dumped, the SQL file is uploaded using the hdfs cli.
+# For AWS or Azure, the url should start with `s3a://` or `abfs://`, respectively.
+
+set -o nounset
+set -o pipefail
+
+LOGFILE=/var/log/dl_postgres_backup.log
+
+echo "Logs at ${LOGFILE}"
+
+exec 3>&1 4>&2
+trap 'exec 2>&4 1>&3' 0 1 2 3
+exec 1> >(tee -a "${LOGFILE}") 2> >(tee -a "${LOGFILE}" >&2)
+
+doLog() {
+  type_of_msg=$(echo "$@" | cut -d" " -f1)
+  msg=$(echo "$*" | cut -d" " -f2-)
+  [[ $type_of_msg == INFO ]] && type_of_msg="INFO " # one space for aligning
+  [[ $type_of_msg == WARN ]] && type_of_msg="WARN " # as well
+
+  # print to the terminal if we have one
+  test -t 1 && echo "$(date "+%Y-%m-%dT%H:%M:%SZ") $type_of_msg ""$msg"
+  echo "$(date "+%Y-%m-%dT%H:%M:%SZ") $type_of_msg ""$msg" >>$LOGFILE
+}
+
+usage() {
```

Comment:
Could we update this method name to be more specific about what this is doing? For example `printInputError()`?